examples/lines_bars_and_markers/scatter_hist.py

```diff
@@ -3,23 +3,63 @@
 Scatter Hist
 ============
 
-Creates histogram from scatter plot
-and adds them to the sides of the plot.
+Show the marginal distributions of a scatter as histograms at the sides of
+the plot.
+
+For a nice alignment of the main axes with the marginals, two options are shown
+below.
+
+* the axes positions are defined in terms of rectangles in figure coordinates
+* the axes positions are defined via a gridspec
+
+An alternative method to produce a similar figure using the ``axes_grid1``
```

Comment:
This logic is a bit screwed. You say "two options are shown", then list two bullet points, then immediately come with a third alternative. Either make it three equal options, or make the semantic hierarchy

- standard figure
  - explicit figure coordinates
  - gridspec
- axes_grid1

more clear.

Or make axes_grid1 an info admonition.

Reply:
This example shows two options. An alternative to those two options is shown in a different example. So clearly, they don't belong into the same hierarchy.

Reply:
I am not sure what the exact wording was when the initial discussion was, but I think that it is currently clear.